pkg/handler/handler.go

```diff
@@ -95,5 +96,6 @@ func New(entities *entities.Entity, logger logger.Logger) *Handler {
 		PkPass:            pkpass.New(entities, logger),
 		Emojis:            emojis.New(entities, logger),
 		Earn:              earn.New(entities, logger),
+		Dex:               dex.New(entities, logger),
```

Comment:
Binance should be a CEX. cmiiw

Reply:
sounds good :lul but let temp use all dex for others too